dev/oracle/18-slim/privileges.sql

```diff
@@ -0,0 +1,27 @@
+-- Copyright (c) 2022, Oracle and/or its affiliates. All rights reserved
+
+create user sequelizetest identified by sequelizepassword;
```

Comment:
Are these different from 21? Maybe we can reference the 21 version of this file instead of copying this

Reply:
This also holds for the other files, try to minimize the copying of files. We want to do that for the other dialects as well later, so this is a good first step

Reply:
I have added a change does this look good?

Reply:
Yes, that's great. Thanks!